src/components/PopupWithForm.js

```diff
@@ -20,6 +21,14 @@ export default class PopupWithForm extends Popup {
     super.close();
```

Comment:
You shouldn’t create the **same methods** of the parent class if you don’t change them inside and only call them with `super`. The parent methods are available in any inheritor of the class.